### DISCUSSION TITLE
Dreambooth extension installation error

### DISCUSSION THREAD

**Discussion by iaovai256** [d8ahazard/sd_dreambooth_extension#975] [Q&A]
Anyone can help pls?

stderr: ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 

'C:\\Users\\iaova\\stable-diffusion-webui\\venv\\Lib\\site-packages\\google\\~rotobuf\\internal\\_api_implementation.cp310-win_amd64.pyd'
Check the permissions

**Comment by iaovai256** [comment]
Other extensions had no problem.

**Comment by Arilziem** [comment]
Delete the tilde (~rotobuf) directory manually, that should resolve the issue. Not sure how or when that happens, but sometimes pip packages are not installed completely. The first character is then replaced with tilde.
